modules/tensor_mechanics/test/tests/czm/tests

```diff
@@ -0,0 +1,144 @@
+[Tests]
+  [./czm_3DC_load_Normal]
+    type = Exodiff
+    input = czm_3DC_load_only_normal.i
+    exodiff = czm_3DC_load_only_normal_out.e
+    issues = '#11546'
+    requirement = 'The system shall allow for cohesive zone laws to'
```

Comment:
The system shall allow for cohesive zone laws to represent the traction-separation behavior at an interface between two bodies represented by continuum elements in 3D using the Salehani Irani 3DC model, and only compute a normal gap under purely normal loading.

Reply:
You can put the information about the expected tractions in a comment in the input file, but I don't think it belongs in the requirement.